js/extension/features/block-sellers.js

```diff
@@ -30,30 +34,33 @@ rl.ready(() => {
    *
    * @method blockSellers
    * @param {String} type Either 'hide' or 'tag'
+   * @param {String} querySelector Either 'marketplaceQuerySelector' or 'wantlistMessageQuerySelector' const
+   * @param {String} tag Either 'a' (anchor) or 'strong' tag
+   * @param {String} clazz Either 'shortcut_navigable' (marketplace) or 'wantlist-card' (wantlist message)
    * @return {function}
    */
 
-  window.blockSellers = function blockSellers(type) {
+  window.blockSellers = function blockSellers(type, querySelector, tag, clazz) {
 
     let _class = type === 'hide' ? 'hidden-seller' : 'blocked-seller';
```

Comment:
Instead of passing in `tag` and `clazz` in each call of `blockSellers` I think it would be cleaner to set those values with an `if/else` statement:

```javascript

    let _class = type === 'hide' ? 'hidden-seller' : 'blocked-seller',
        tag,
        clazz;

    if (querySelector === marketplaceQuerySelector) {
      tag = 'a'
      clazz = '.shortcut_navigable'
    } else {
      tag = 'strong'
      clazz = '.wantlist-card'
    }
```